espflash/src/bin/espflash.rs

```diff
@@ -30,7 +30,7 @@ pub struct Cli {
     subcommand: Commands,
 
     /// Do not check for updates
-    #[clap(short, long, global = true, action)]
+    #[clap(long, global = true, action)]
```

Comment:
I guess it clashed with `flash_size`, maybe we could use another short instead of removing it
```suggestion
    #[clap(short = 'S', long, global = true, action)]
```